FlatGrid.js

```diff
@@ -157,34 +158,37 @@ const FlatGrid = memo(
       [keyExtractor],
     );
 
+    const customInterfaceProps = {
```

Comment:
You wont need to extract these outside with above implementation - it would be just one component used below.